app/views/settings/dex_connector_ldaps/_form.html.slim

```diff
@@ -53,14 +53,17 @@
     = error_messages_for(@certificate_holder, :bind_anon)
 
   .ldap-dn-authentication.collapse class="#{'in' if !@certificate_holder.bind_anon}" id="anonymous_bind_toggle" aria-expanded="true"
-    .form-group class="#{error_class_for(@certificate_holder, :bind_dn)}"
+    .form-group class="#{error_class_for(@certificate_holder, :bind_dn)}" autocomplete="nope"
```

Comment:
There's no advantage on putting this in the `.form-group` div element. I suggest you to move it directly to the input text field element.

Reply:
I also read about adding an invalid option to enforce non-autocompletion but a simply `off` would do the job, no? I'm not aware of the unexpected behavior scenarios. Did you face any?

Reply:
Using "off" is not sufficient. It works in Chrome to fix the problem but not Firefox. The extra field seems unnecessary but we determined it is needed to correct the bug.